R/DepthProfile.R

```diff
@@ -0,0 +1,1407 @@
+#' TADA_DepthCategory.Flag
```

Comment:
Recommend to try to use similar function naming convention as other TADA functions if possible. I don't think any TADA functions currently include a "." in the name.

How about:
TADA_FlagDepthCategory

Reply:
I will make that change for consistency.